### PR TITLE
Remove pending tests

### PR DESCRIPTION
**Pull request by ricardopereira**
#541

### CODE REVIEW

File: Spec/Auth.swift [hunk 1]
```diff
@@ -2620,104 +2620,84 @@ class Auth : QuickSpec {
...
 
-                // get first token
-                let tokenParams = ARTTokenParams()
-                tokenParams.capability = "{\"wrongchannel\": [\"*\"]}"
-                tokenParams.clientId = "testClientId"
+        describe("Reauth") {
 
-                var firstToken = ""
+            // RTC8
+            it("should use authorise({force: true}) to reauth with a token with a different set of capabilities") {
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: force: true is no longer supported

File: Spec/Auth.swift [hunk 2]
```diff
@@ -2620,104 +2620,84 @@ class Auth : QuickSpec {
...
                 }
-                expect(firstToken).toNot(beNil())
-                expect(firstToken.characters.count > 0).to(beTrue())
 
-                // init ARTRealtime
-                let realtimeOptions = AblyTests.commonAppSetup()
-                realtimeOptions.token = firstToken
-                realtimeOptions.clientId = "testClientId"
+                let authOptions = ARTAuthOptions()
+                authOptions.force = true
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: force is not supported?

File: Spec/Auth.swift [hunk 3]
```diff
@@ -2620,104 +2620,84 @@ class Auth : QuickSpec {
...
+                    realtime.auth.authorise(tokenParams, options: authOptions) { tokenDetails, error in
+                        expect(error).to(beNil())
+                        expect(tokenDetails).toNot(beNil())
                         done()
                     }
-                    realtime.connect()
                 }
 
-                // create a `rightchannel` channel and check can't attach to it
-                let channel = realtime.channels.get("rightchannel")
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: Why did you remove this?  This is the point of the test no, i.e. attach with one set of permissions, change them, check that permissions have actually changed by checking the channel attach success?

File: Spec/Auth.swift (hunk 1 above)
Comment (mattheworiordan) [inline Spec/Auth.swift]: Ok, but I'm not sure why we are doing any work on `0.8` now other than critical fixes?

**Comment by mattheworiordan** [comment]
👍 for merging.

**Comment by ricardopereira** [comment]
I still need to see why the tests are no passing 🙁

**Comment by mattheworiordan** [comment]
> I still need to see why the tests are not passing 🙁

Any update?

File: Spec/Auth.swift (hunk 1 above)
Comment (ricardopereira) [inline Spec/Auth.swift]: Ok, I agree.

File: Spec/Auth.swift (hunk 3 above)
Comment (ricardopereira) [inline Spec/Auth.swift]: I will rollback 👍

**Comment by ricardopereira** [comment]
I'm finishing this now.
